src/cli/subscriptions.ts

```diff
@@ -27,4 +27,14 @@ require('yargs')
       describe: 'Path to exported credentials file',
       type: 'string',
     },
+
+    'pubsub-server-port': {
```

Comment:
Makes sense to prefix env vars with PUBSUB but no need in cli args
```suggestion
    'server-port': {
```